test cases/osx/2 library versions/meson.build

```diff
@@ -1,6 +1,12 @@
 project('library versions', 'c')
 
+zlib_dep = dependency('zlib')
+
 some = shared_library('some', 'lib.c',
+  # duplicate the rpath again, in order
+  # to test Meson's RPATH deduplication
+  build_rpath : zlib_dep.get_pkgconfig_variable('libdir'),
+  dependencies : zlib_dep,
```

Comment:
@nirbheek Without the fix, I get failures:
https://travis-ci.org/mesonbuild/meson/jobs/421642823
https://travis-ci.org/mesonbuild/meson/jobs/421642824